api/src/integration/resources/data.sql

```diff
@@ -0,0 +1,5 @@
+insert into config (config_id, configuration) values
```

Comment:
Add comment describing how this gets used / where it's referenced from?

Reply:
Done.

Reply:
OK. There's no direct reference to these files in ApplicationTest... can you describe the mechanism that causes them to get loaded?